src/query/processor/cypher/astbuilder/ASTBuilder.cpp

```diff
@@ -0,0 +1,1701 @@
+/**
+Copyright 2024 JasmineGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+#include "ASTBuilder.h"
+#include <any>
+#include <string>
+#include "ASTInternalNode.h"
+#include "ASTLeafNoValue.h"
+#include "ASTLeafValue.h"
+#include "/home/ubuntu/software/jasminegraph/code_generated/antlr/CypherBaseVisitor.h"
+
+any visitOC_Cypher(CypherParser::OC_CypherContext *ctx) ;
```

Comment:
Can we reorder the appearance of the function definitions in alphabetical order of the letters appearing after _. For example `visitOC_Cypher` we should use the letter `C` and the `y` if needed for alphabetical sorting.

Reply:
the order is very similar to hierarchy of the cypher grammar rule set. and this order was defined by antlr4 then it can be helpful for debugging. if it is necessary, I can re-order it.

Reply:
In this case lets not re-order the functions.

Reply:
okay sir.